doc/api/errors.md

```diff
@@ -1847,20 +1847,25 @@ Creation of a [`zlib`][] object failed due to incorrect configuration.
 A module file could not be resolved while attempting a [`require()`][] or
 `import` operation.
 
+<a id="legacy-nodejs-error-codes"></a>
```

Comment:
I may be wrong, but we may need not this hardcoded id, as the other ones in this doc may be due to legacy links support. Let us see what others think.